app/src/store/actionCreators.js

```diff
@@ -0,0 +1,81 @@
+import { actionTypes } from './actionTypes';
+
+const globalUrl = 'http://localhost:4000';
+
+function getMovies(dispatch, getState){
+    var url = globalUrl + '/movies';

```

Comment:
1. forget `var` plz

2. we use apiMap file - where we store all functions which returns urls for out api

Reply:
Done: replaced `var` to `let`, added `apiMap` file with urls